Source/Scene/ModelExperimental/ModelExperimental3DTileContent.js

```diff
@@ -173,6 +173,7 @@ ModelExperimental3DTileContent.prototype.update = function (
   model.colorBlendAmount = tileset.colorBlendAmount;
```

Comment:
Commenting on code outside the diff...

I don't think we want `ModelExperimental3DTileContent.prototype.applyStyle` to set the style directly. I think it should continue to call `applyStyle`. Also the `ModelExperimental` style setter should early exit if it sees the same style. I think this leads to the more consistent behavior. Say someone writes an application that sets `model.style = myStyle` every frame, we wouldn't want to regenerate commands every frame.

Reply:
I pushed a commit with this change